01/app.js

```diff
@@ -1 +1,11 @@
-console.log('DOM');
\ No newline at end of file
+
+const liElement = document.querySelector('.comments__item.comments__item--newest');
+const dataInfoList = liElement.querySelectorAll('[data-info]');
+
+if(liElement && dataInfoList) {
+    console.log(`Znaleziono ${dataInfoList.length} poniższe elementy z atrybutem  data-info :`);
```

Comment:
Tutaj mamy drobny błąd logiczny - sprawdzasz czy element został wyszukany później niż go wykorzystujesz.
Powinno być:
```
const liElement = document.querySelector('.comments__item.comments__item--newest');
if(liElement) {
    const dataInfoList = liElement.querySelectorAll('[data-info]');
    console.log(`Znaleziono ${dataInfoList.length} poniższe elementy z atrybutem  data-info :`);
}
```

Nie trzeba sprawdzać `dataInfoList` bo to tablica elementów więc nieważne czy jest ich 0 czy więcej to błędu nie będzie.